src/main/cpp/blaze.cc

```diff
@@ -432,6 +432,12 @@ static vector<string> GetServerExeArgs(const blaze_util::Path &jvm_path,
   // https://github.com/openjdk/jdk/blob/2faf8b8d582183275b1fdc92313a1c63c1753e80/src/java.base/share/classes/sun/nio/fs/AbstractWatchKey.java#L40
   result.push_back("-Djdk.nio.file.WatchService.maxEventsPerPoll=10000");
 
+#if defined(_WIN32)
+  // See and use more than 64 CPUs on Windows.
+  // https://bugs.openjdk.org/browse/JDK-6942632
+  result.push_back("-XX:+UseAllWindowsProcessorGroups");
```

Comment:
Not sure if we can also check if the JDK supports this flag or not? If some user runs bazel_nojdk with a local JDK that doesn't support this flag, I assume it would fail?

Reply:
I made this flag optional.

Do we have a policy regarding what we expect of a custom JDK? Assuming the latest patch version that is generally available seems reasonable to me, but only if stated explicitly.

Reply:
It should be optional now.

Reply:
I don't think we have a written-down policy for this, just want minimize the breaking change for users. This looks good, thanks!